### ISSUE TITLE
v3.22.1 SHA256 hash not valid

### ISSUE THREAD

**Issue by codygreen** [F5Networks/f5-appsvcs-templates#22]
### Environment
 * FAST RPM Version: 3.22.1
 * BIG-IP Version: any version

### Summary
The hash validation for version 3.22.1 will fail since the hash is for: dist/f5-appsvcs-templates-1.3.0-1.noarch.rpm

This needs to be the filename without a directory: f5-appsvcs-templates-1.3.0-1.noarch.rpm

### Steps To Reproduce
List the steps to reproduce the behavior:
Run ansible role:
```python
---
# file: roles/bigips/tasks/atc_install.yml
- name: Download and Install FAST RPM sha check - url
  include_role:
    name: f5devcentral.f5app_services_package
  vars:
    f5app_services_package_url: https://github.com/F5Networks/f5-appsvcs-templates/releases/download/v{{ fast_version[:-2] }}/f5-appsvcs-templates-{{ fast_version }}.noarch.rpm
    f5app_services_package_checksum_url: https://github.com/F5Networks/f5-appsvcs-templates/releases/download/v{{ fast_version[:-2] }}/f5-appsvcs-templates-{{ fast_version }}.noarch.rpm.sha256
    f5app_services_package_path: /tmp/f5-appsvcs-templates-{{ fast_version }}.noarch.rpm
```

### Expected Behavior
The RPM hash validation should succeed

### Actual Behavior
The RPM hash validation is failing:
```
fatal: [bigip1]: FAILED! => {"changed": false, "msg": "Unable to find a checksum for file 'f5-appsvcs-templates-1.3.0-1.noarch.rpm' in 'https://github.com/F5Networks/f5-appsvcs-templates/releases/download/v1.3.0/f5-appsvcs-templates-1.3.0-1.noarch.rpm.sha256'"}
```

**Comment by zinkem5** [comment]
Cody, can this be reproduced without Ansible?

**Comment by emmanuelgirard** [comment]
@zinkem5 
I can confirm same error directly from bash : 
```bash
sha256sum f5-appsvcs-templates-1.3.0-1.noarch.rpm -c f5-appsvcs-templates-1.3.0-1.noarch.rpm.sha256
sha256sum: f5-appsvcs-templates-1.3.0-1.noarch.rpm: no properly formatted SHA256 checksum lines found
sha256sum: dist/f5-appsvcs-templates-1.3.0-1.noarch.rpm: No such file or directory
dist/f5-appsvcs-templates-1.3.0-1.noarch.rpm: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
```

**Comment by delgadillo22** [comment]
Thanks for the report. I've created MYSTIQUE-230 to track this.

**Comment by zinkem5** [comment]
Closing this one, as it has been fixed.

**Comment by mirzawaqasahmed** [comment]
Re-opening this issue...
While installing `f5-appsvcs-templates-1.4.0-1.noarch.rpm`
getting the following error:

```
Package installation failed with error: Task: /mgmt/shared/iapp/package-management-tasks/9dca4bec-18f5-46ed-b833-ed89c5b205b4 failed to complete, with error message: Failed to install /var/config/rest/downloads/f5-appsvcs-templates-1.4.0-1.noarch.rpm - error: unpacking of archive failed on file /var/config/rest/iapps/f5-appsvcs-templates/lib/drivers.js;5f9b2e83: cpio: MD5 sum mismatch
```

**Comment by shyawnkarim** [comment]
This issue was resolved with [Release 1.4.0](https://github.com/F5Networks/f5-appsvcs-templates/releases/tag/v1.4.0).
